source/features/index.tsx

```diff
@@ -150,6 +157,29 @@ const setupPageLoad = async (id: FeatureID, config: InternalRunConfig): Promise<
 	}
 };
 
+// Fetch hotfix releases asynchronously
+// every 6 hours ( 4 times a day )
+// this is achieved by GETting a JSON file
+// from the hotfix branch and comparing
+// with the current version of extension
+const fetchHotfixesAsynchronously = cache.function(async () => {
+	const response = await fetch('https://raw.githubusercontent.com/sindresorhus/refined-github/hotfix/hotfix.json');
+	const hotfixes = await response.json();
+
+	if (!hotfixes) {
+		return {};
+	}
+
+	if (hotfixes.unaffected) {
+		const currentVersion = browser.runtime.getManifest().version;
+		if (looseVersionCompare(hotfixes.unaffected, currentVersion) >= 0) {
```

Comment:
I think this is the correct condition
```suggestion
		if (looseVersionCompare(hotfixes.unaffected, currentVersion) < 1) {
```

Reply:
I pondered over this for a while before deciding it was accurate. I believe we are skipping the application of the hotfix with `return {}` here.

Hence, it would make sense that when the `versionCompare` check is `-1` ( second one is greater ), it should pass-through.

Reply:
wait, I just realized that my assumption was wrong. I was looking at the code example in the [Usage section](https://github.com/fregante/tiny-version-compare#usage) of the `README` for `tiny-version-compare`; but the [API section](https://github.com/fregante/tiny-version-compare#api) suggests the opposite as in the review here.

I'll put up a pull request for that change to that repository as well :)

Reply:
Nope.

```js
tinyVersionCompare('20.9.4', '20.9.3') === 1 // Is affected, don't return {}
tinyVersionCompare('20.9.4', '20.9.4') === 0 // Is NOT affected, exit
tinyVersionCompare('20.9.4', '20.9.5') === -1 // Is NOT affected, exit
```

The examples are correct. Run them yourself on RunKit:

https://npm.runkit.com/tiny-version-compare

<img width="536" alt="Screen Shot 2020-09-05 at 09 37 41" src="https://user-images.githubusercontent.com/1402241/92307356-74459100-ef5b-11ea-9de4-8f3e7f31a45d.png">

Reply:
> but the [API section](https://github.com/fregante/tiny-version-compare#api) suggests the opposite as in the review here

Uh, yeah. In my mind -1 meant left one is greater, 1 meant right is greater, hence the confusion 🤦

Reply:
I fixed the readme

Reply:
> In my mind -1 meant left one is greater, 1 meant right is greater

yes, that makes logical sense :
```
   -1		 0		1
  left	  equal  right
```

this means that this suggestion is incorrect and the original version was accurate right?

Reply:
My mind was incorrect, the suggestion is incorrect. Test it.